app/assets/javascripts/analytics/scroll-tracker.js

```diff
@@ -402,7 +402,7 @@
       ['Percent', 100]
     ],
     '/eubusiness': [
-      ['Heading', 'Additional help and support']
+      ['Heading', 'Webinars for EU-based businesses trading with the UK']
     ]
```

Comment:
To extend it to the translated pages, do we need to make the scroll tracking based on percentage rather than heading?

Reply:
For posterity, we're going to switch to a percentage-based measurement in a follow-up PR